group_vars/openstack/slurm

```diff
@@ -9,6 +9,7 @@ slurm_login:
   root_volume_size: 0
   node_resource: "Cluster::Node" 
   nodenet_resource: "Cluster::NodeNet1"
+  user: centos
```

Comment:
can you please change this to `{{ centos_deploy_user }}`

Reply:
Changed to `user: "{{ cluster_deploy_user }}"`